mutation/range_tombstone_change_generator.hh

```diff
@@ -141,6 +143,10 @@ public:
     }
 
     void consume(range_tombstone rt) {
+        position_in_partition::tri_compare cmp(_schema);
+        if (cmp(rt.end_position(), _lower_bound) <= 0) {
```

Comment:
Key comparison is relatively expensive, the validator has it disabled by default. Do we really need this?

Reply:
Good point, it's just a sanity check.
I'll see if it could be enabled based on the validator settings